### PR TITLE
Info cards on homepage

### PR DESCRIPTION
**Pull request by sennett**
# [Info cards on homepage](https://app.shortcut.com/oazo-apps/story/3228/create-homepage)
  
## Changes 👷‍♀️
- adds info cards to homepage (also mobile view)
- removes unnecessary ProductCardLayout component
- fixes text wrapping issue when Product description is long on product cards:
  
## How to test 🧪
- open homepage
- observe the info cards at the bottom
- resize the viewport and make sure they collapse properly
    
## Definition of done ✔️

- [ ] Acceptance criteria for each issue met
- [ ] Unit tests written where needed and passing
- [ ] End-to-end tests for happy path
- [ ] Documentation updated <When applicable>
- [ ] Project builds without errors
- [ ] Non-functional requirements met
- [ ] Code reviewed and functionality tested by the reviewer (locally, Heroku, etc.)
- [ ] Feature verified and accepted by Product Owner
- [ ] Project deployed to production environment

### CODE REVIEW

**Comment by piekczyk** [comment]
Small shape updated will be needed according to design. Info cards should be ~square on the mobile. 
image

File: components/InfoCard.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    >
+      <Box sx={{ position: 'relative', zIndex: 2 }}>
+        <Heading sx={{ my: 2, fontWeight: 'bold', color: 'primary' }}>{title}</Heading>
+        <Text sx={{ mb: 3, color: 'text.subtitle', minHeight: '3em' }}>{subtitle}</Text>
+        {links.map(({ href, text }, i) => (
+          <>
+            <AppLink
+              key={i}
+              href={href}
+              sx={{ pb: 3, fontSize: '16px', color: 'primary', display: 'block' }}
```
Comment (piekczyk) [inline components/InfoCard.tsx:42]: fontSize: 3

File: components/InfoCard.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        backgroundRepeat: 'no-repeat',
+      }}
+    >
+      <Box sx={{ position: 'relative', zIndex: 2 }}>
+        <Heading sx={{ my: 2, fontWeight: 'bold', color: 'primary' }}>{title}</Heading>
+        <Text sx={{ mb: 3, color: 'text.subtitle', minHeight: '3em' }}>{subtitle}</Text>
+        {links.map(({ href, text }, i) => (
+          <>
+            <AppLink
+              key={i}
```
Comment (piekczyk) [inline components/InfoCard.tsx:40]: using array key is not recommended but since links array is static it can stay like this

**Review by piekczyk — CHANGES_REQUESTED** [review]

File: components/InfoCard.tsx (hunk 2 above)
Comment (sennett) [reply components/InfoCard.tsx]: I thought that all mapping react components like this needed a key?  is the issue that the key is not on the <>?  I removed the unnecessary fragment in either case.

**Review by piekczyk — APPROVED** [review]

**Pull request merged**